diff --git a/scripts/linux/4_boot_leds.sh b/scripts/linux/4_boot_leds.sh
index 053e021..da1e24f 100755
--- a/scripts/linux/4_boot_leds.sh
+++ b/scripts/linux/4_boot_leds.sh
@@ -1,4 +1,7 @@
 #!/bin/bash
 
-rm -rf /home/xilinx/pynq/bitstream/.log
-python3.4 /home/xilinx/scripts/boot.py &
+if [ -f /etc/profile.d/python3.6.sh ]; then
+  source /etc/profile.d/python3.6.sh
+fi
+
+python3.6 /home/xilinx/scripts/boot.py &
