diff --git a/scripts/linux/3_pl_server.sh b/scripts/linux/3_pl_server.sh
index 23cae6e..43e6f3c 100755
--- a/scripts/linux/3_pl_server.sh
+++ b/scripts/linux/3_pl_server.sh
@@ -1,7 +1,11 @@
 #!/bin/bash
 
+if [ -f /etc/profile.d/python3.6.sh ]; then
+  source /etc/profile.d/python3.6.sh
+fi
+
 rm -rf /home/xilinx/pynq/bitstream/.log
-python3.4 /home/xilinx/scripts/start_pl_server.py &
+python3.6 /home/xilinx/scripts/start_pl_server.py &
 
 script_name=`readlink -f "$0"`
 
